cairo/ethereum/cancun/keth/body.cairo

```diff
@@ -116,31 +116,14 @@ func body{
     finalize_state{state=state}();
     BlockEnvImpl.set_state{block_env=block_env}(state);
 
-    // Squash the receipts, transactions, and withdrawals dicts once they're no longer being modified.
```

Comment:
why remove the squashing?
Is it done before?

Reply:
it's done in the block output commitment